src/awscdk/awscdk-app-java.ts

```diff
@@ -1,32 +1,17 @@
 import { join } from 'path';
-import * as semver from 'semver';
-import { SampleDir } from '..';
+import { AwsCdkDepsJava } from '.';
+import { DependencyType, SampleDir } from '..';
 import { JavaProject, JavaProjectOptions } from '../java';
+import { AwsCdkDeps, AwsCdkDepsCommonOptions } from './awscdk-deps';
 import { CdkConfig, CdkConfigCommonOptions } from './cdk-config';
 import { CdkTasks } from './cdk-tasks';
 
-export interface AwsCdkJavaAppOptions extends JavaProjectOptions, CdkConfigCommonOptions {
-  /**
-   * AWS CDK version to use (you can use semantic versioning).
-   *
-   * @default "^1.130.0"
-   */
-  readonly cdkVersion: string;
-
-  /**
-   * Which AWS CDK modules this app uses. The `core` module is included by
-   * default and you can add additional modules here by stating only the package
-   * name (e.g. `aws-lambda`).
-   *
-   * @featured
-   */
-  readonly cdkDependencies?: string[];
-
+export interface AwsCdkJavaAppOptions extends JavaProjectOptions, CdkConfigCommonOptions, AwsCdkDepsCommonOptions {
   /**
    * The name of the Java class with the static `main()` method. This method
    * should call `app.synth()` on the CDK app.
    *
-   * @default "org.acme.App"
+   * @default "org.acme.CdkApp"
```

Comment:
Lets call it `MyApp`:

```suggestion
   * @default "org.acme.MyApp"
```